modules/gapi/src/backends/onnx/gonnxbackend.cpp

```diff
@@ -471,6 +469,8 @@ inline Ort::Value createTensor(const Ort::MemoryInfo& memory_info,
         return createTensor<float>(memory_info, tensor_params, data);
     case ONNX_TENSOR_ELEMENT_DATA_TYPE_INT32:
         return createTensor<int32_t>(memory_info, tensor_params, data);
+    case ONNX_TENSOR_ELEMENT_DATA_TYPE_INT64:
```

Comment:
Ok, probably worth comment that conversion from to i64 is done later

Reply:
ok will do